components/App.vue

```diff
@@ -133,7 +133,7 @@ export default {
       this.networkConfig = deployConfig[this.activeNetwork];
       if (this.networkConfig) {
         const {contract, ketherNFT, ketherView} = loadContracts(this.networkConfig, this.provider)
-        this.setContracts(contract, ketherNFT, ketherView);
+        await this.setContracts(this.activeNetwork, contract, ketherNFT, ketherView);
```

Comment:
fwiw `setContracts` can get the activeNetwork from the contract but I wasn't sure if it would give us an extra network call.